packages/server/lib/controllers/appAuth.controller.ts

```diff
@@ -125,36 +125,19 @@ class AppAuthController {
                 return;
             }
 
-            const { success, error, response: credentials } = await connectionService.getAppCredentials(provider, config, connectionConfig);
-
-            if (!success || !credentials) {
-                void logCtx.error('Error during app token retrieval call', { error });
+            const credentialsRes = await githubAppClient.createCredentials({ provider: provider as ProviderGithubApp, integration: config, connectionConfig });
+            if (credentialsRes.isErr()) {
+                report(credentialsRes.error);
+                void logCtx.error('Error during Github App credentials creation', { error: credentialsRes.error });
                 await logCtx.failed();
-
-                void connectionCreationFailedHook(
```

Comment:
Same as apple store refactor, it's probably good that we have that but it's inconsistent across the endpoints and even in this endpoint